### PR TITLE
(partial?) fix for e2e tests

### PR DESCRIPTION
**Pull request by levb**
Fixed passing the bundle into the e2e tests.

@hanzei would be great if you could add running `make test-e2e` to ci? What happens if we just include it in `make all`?

### CODE REVIEW

**Review by catalintomai — APPROVED** [review]

**Review by hanzei — CHANGES_REQUESTED** [review]
It seems like we still need `MM_SERVER_PATH` as `TranslationsPreInit()` otherwise doesn't know where to look for the translations files and fails with: 
```
--- FAIL: TestPPAPI (0.47s)
panic: unable to load Mattermost translation files: Failed to load system translations for 'en' [recovered]
	panic: unable to load Mattermost translation files: Failed to load system translations for 'en'

goroutine 116 [running]:
testing.tRunner.func1.2(0x29153c0, 0xc000500048)
	/usr/lib/go/src/testing/testing.go:1144 +0x49f
testing.tRunner.func1(0xc000603080)
	/usr/lib/go/src/testing/testing.go:1147 +0x695
panic(0x29153c0, 0xc000500048)
	/usr/lib/go/src/runtime/panic.go:971 +0x499
github.com/mattermost/mattermost-server/v5/api4.setupTestHelper(0x3538888, 0xc0004ab000, 0xc0000b05a0, 0xc000780100, 0x0, 0xc000080800)
	/home/sumacheb/go/pkg/mod/github.com/mattermost/mattermost-server/v5@v5.3.2-0.20200929105438-ed2cd0e55219/api4/apitestlib.go:100 +0x2105
github.com/mattermost/mattermost-server/v5/api4.Setup(0x352fa20, 0xc000603080, 0x41c97c0)
	/home/sumacheb/go/pkg/mod/github.com/mattermost/mattermost-server/v5@v5.3.2-0.20200929105438-ed2cd0e55219/api4/apitestlib.go:213 +0x1f2
github.com/mattermost/mattermost-plugin-apps/server/http/restapi.Setup(0x352fa20, 0xc000603080, 0x0)
	/home/sumacheb/src/mattermost/plugins/apps/server/http/restapi/restapitestlib.go:45 +0xb3
github.com/mattermost/mattermost-plugin-apps/server/http/restapi.TestPPAPI(0xc000603080)
	/home/sumacheb/src/mattermost/plugins/apps/server/http/restapi/restapi_test.go:17 +0x65
testing.tRunner(0xc000603080, 0x2b93688)
	/usr/lib/go/src/testing/testing.go:1194 +0x203
created by testing.(*T).Run
	/usr/lib/go/src/testing/testing.go:1239 +0x5d8
```

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,8 @@ dist/
 
 # Jetbrains
 .idea/
+
+# e2e test aux output
+/server/http/restapi/mattermost.log
+/server/http/restapi/notifications.log
+/server/http/restapi/data/
```
Comment (hanzei) [inline .gitignore:15]: Nit pick: Add empty line

File: Makefile [hunk 2]
```diff
@@ -247,9 +247,9 @@ ifneq ($(HAS_WEBAPP),)
 endif
 
 .PHONY: test-e2e
-test-e2e:
+test-e2e: dist
 	@echo Running e2e tests
-	MM_SERVER_PATH=${MM_SERVER_PATH} $(GO) test -v $(GO_TEST_FLAGS) -tags=e2e $(GO_PACKAGES)
+	PLUGIN_BUNDLE=$(shell pwd)/dist/$(BUNDLE_NAME) MM_SERVER_PATH=${MM_SERVER_PATH} $(GO) test -v $(GO_TEST_FLAGS) -tags=e2e $(GO_PACKAGES)
```
Comment (hanzei) [inline Makefile:252]: Do we need to use `GO_PACKAGES` or can we fall back to `./...`?

**Comment by hanzei** [comment]
@levb I've filled https://mattermost.atlassian.net/browse/MM-33340 and will bring that ticket up in Sprint planing.

**Comment by levb** [comment]
> It seems like we still need `MM_SERVER_PATH`

I thought I didn't change the setting of this env variable, am I not understanding something? @hanzei How do I reproduce that crash?

File: Makefile (hunk 2 above)
Comment (levb) [reply Makefile:252]: not sure, @catalintomai? But for this PR - I did not change that logic.

**Comment by hanzei** [comment]
@levb Steps to repo:

1. Make sure `MM_SERVER_PATH` is not set
2. Run `make test-e2e`
3. Observe panic

AFAIK `MM_SERVER_PATH` is not set by default anywhere in this project nor in the server.

File: Makefile (hunk 2 above)
Comment (catalintomai) [reply Makefile:252]: 0/5 - GO_PACKAGES  is already used in the MM codebase (and amounts to the same)

Comment (hanzei) [inline Makefile:252]: Just tested: The command works if `MM_SERVER_PATH` is set, which seems fine as long the test given a better error message.

Comment (hanzei) [inline Makefile:252]: Why is `MM_SERVER_PATH` is passed to the command again? Is the env variable not enough?

**Comment by levb** [comment]
> 1. Make sure `MM_SERVER_PATH` is not set
> 2. Run `make test-e2e`
> 3. Observe panic

Works for me, even with `MM_SERVER_PATH=` removed from the command, no panic

**Pull request closed without merging**